rasa/core/actions/forms.py

```diff
@@ -584,6 +585,28 @@ async def activate(
         # collect values of required slots filled before activation
         prefilled_slots = {}
 
+        # run slot extraction the second time during form activation
+        # for slots with mapping conditions
+        action_extract_slots = action.action_for_name_or_text(
+            ACTION_EXTRACT_SLOTS, domain, self.action_endpoint
+        )
+
+        logger.debug(
+            f"Executing default action '{ACTION_EXTRACT_SLOTS}' at form activation."
+        )
+
+        extraction_events = await action_extract_slots.run(
+            output_channel, nlg, tracker, domain
+        )
+
+        events_as_str = "\n".join([str(e) for e in extraction_events])
+        logger.debug(
+            f"The execution of '{ACTION_EXTRACT_SLOTS}' resulted in "
+            f"these events: {events_as_str}."
+        )
```

Comment:
Since the tracker that's used effectively is a temporary tracker during `action.run()`, it doesn't matter much that the log gets issued after this update. I will explain in your other comment the decision behind this temp tracker.